Repository/ServiceEntityRepositoryProxy.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Bundle\DoctrineBundle\Repository;
+
+use Doctrine\Common\Collections\AbstractLazyCollection;
+use Doctrine\Common\Collections\Criteria;
+use Doctrine\Common\Collections\Selectable;
+use Doctrine\DBAL\LockMode;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Doctrine\ORM\Query\ResultSetMappingBuilder;
+use Doctrine\ORM\QueryBuilder;
+use Doctrine\Persistence\ManagerRegistry;
+use LogicException;
+use Symfony\Component\VarExporter\LazyObjectInterface;
+
+use function sprintf;
+
+/**
+ * Optional EntityRepository base class with a simplified constructor (for autowiring).
+ *
+ * To use in your class, inject the "registry" service and call
+ * the parent constructor. For example:
+ *
+ * class YourEntityRepository extends ServiceEntityRepository
+ * {
+ *     public function __construct(ManagerRegistry $registry)
+ *     {
+ *         parent::__construct($registry, YourEntity::class);
+ *     }
+ * }
```

Comment:
I think this was c&p'd and should be here for this internal class

Reply:
I've moved those doc blocks to the `ServiceEntityRepository` class because that's that class people should actually extend.